R/dash.R

```diff
@@ -370,8 +377,32 @@ Dash <- R6::R6Class(
     # private fields defined on initiation
     name = NULL,
     serve_locally = NULL,
-    routes_pathname_prefix = NULL,
-    requests_pathname_prefix = NULL,
+    routes_pathname_prefix = {
```

Comment:
This syntax is interesting - is this like a function that gets called whenever this variable is accessed? Or is it just a function shorthand?

Reply:
This is just some logic to set the initial value, it’s on the private side, so cannot be modified directly from the console.

Reply:
I realized this isn't going to work the way we want, so I've :hammer: a bit, hopefully the refactored implementation of the routes/requests pathname prefixes is closer to the Python logic.